.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "singleQuote": true
```

Comment:
our codebase for the other SDKs and tools are not using `;` I don't have a strong opinion about it but maybe we should keep the same behaviour

Reply:
Yepp - that's true, it's just a prettier setting .. I don't mind changing it

Reply:
(I am just used to semicolons 🤓)